### PR TITLE
IS-13764: Documented $based_on_properties and fixed an error

### PR DESCRIPTION
**Pull request by grove**

### CODE REVIEW

File: hub/documentation/data-synchronization/connectors/contract-connectors.rst [hunk 1]
```diff
@@ -93,7 +93,11 @@ Properties
    * - ``$based_on``
      - Dict<String,Any>
      - The entity must contain the ``$based_on`` property, which should be a dict with property value pairs for all system raw properties that should be written back to the system. The ``$based_on`` property should be a subset of the properties from the connector output dataset as it should include the property values that was collected for this particular version of the entity. The property will be used to check if there has been modifications made to the entity in the target system after it was collected. The property can be used for preventing lost writes and avoiding unneccesary updates.
-     - Required during insert. Optional for updated and deletes.
+     - Required for updates and deletes. Disallowed for insert.
+   * - ``$based_on_properties``
+     - List<String>
+     - This is property enumerates the properties that are considered for ``$based_on`` comparison. The string values can be property names or a path of properties separated by a ``.`` character.
```
Comment (mMaydew) [inline hub/documentation/data-synchronization/connectors/contract-connectors.rst:99]: Can drop the "is" in the beginning

File: hub/documentation/data-synchronization/connectors/contract-connectors.rst [hunk 2]
```diff
@@ -93,7 +93,11 @@ Properties
...
+     - Required for updates and deletes. Disallowed for insert.
```
Comment (mMaydew) [inline hub/documentation/data-synchronization/connectors/contract-connectors.rst:100]: Dupe of 96 or?

File: hub/documentation/data-synchronization/connectors/contract-connectors.rst (hunk 1 above)
Comment (grove) [reply hub/documentation/data-synchronization/connectors/contract-connectors.rst:99]: Agree. Will fix

File: hub/documentation/data-synchronization/connectors/contract-connectors.rst (hunk 2 above)
Comment (grove) [reply hub/documentation/data-synchronization/connectors/contract-connectors.rst:100]: Nope, just same rules for the two properties.

**Review by mMaydew — APPROVED** [review]
LGTM!

**Pull request merged**